README.md

```diff
@@ -88,11 +88,23 @@ Or you can build your own image:
 
 ```bash
 $ docker build . -t gccrs-dev
+```
+If you want to build an object file: 
+
+```bash
 $ docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp \
     gccrs-dev:latest gccrs -g -O2 -c \
     gcc/testsuite/rust.test/compilable/type_infer1.rs -o type_infer1.o
 ```
 
+If you want to build an executable file:
+```bash
+$ docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp \
+    gccrs-dev:latest gccrs -g -O2 \
+    gcc/testsuite/rust.test/compilable/type_infer1.rs -o type_infer1.o
```

Comment:
I think in terms of compiling an example to an executable you should remove the .o on the output location so it does not confused as an object file

Reply:
Yes , you are right ! I will make the change